config.json

```diff
@@ -748,7 +754,14 @@
         "name": "Pythagorean Triplet",
         "uuid": "394755a3-c743-4b85-b9b8-387907f4e32d",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "classes",
+          "arrays",
+          "for-loops",
+          "numbers",
+          "math",
+          "conditionals"
+        ],
```

Comment:
Here is my solution : https://exercism.org/tracks/javascript/exercises/pythagorean-triplet/solutions/TomPradat

By the way I see a lot of exercises like this one using classes which is a pretty high level concept in the JS track. Should we reevaluate the use of classes in this exercise ? Not add the `classes` as a prerequisites ? Leave it as is ?

Reply:
It's a good question. I think it's fine for now, we'll reevaluate together after if you want.